UPGRADING.md

```diff
@@ -0,0 +1,14 @@
+# Upgrade Guide
+
+This document describes breaking changes and how to upgrade. For a complete list of changes including minor and patch releases, please refer to the [changelog](CHANGELOG.md).
+
+## v2
+
+No major changes to the API.
```

Comment:
Technically that's true, but we shouldn't sell ourselves short :)

Perhaps we can mention that level supports Promises now?